Index: paper_code_aluminum/Graph_code.ipynb
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/paper_code_aluminum/Graph_code.ipynb b/paper_code_aluminum/Graph_code.ipynb
--- a/paper_code_aluminum/Graph_code.ipynb	(revision b1065dd120cc4a9ccf0394348e79d07fc26cc293)
+++ b/paper_code_aluminum/Graph_code.ipynb	(date 1631201874784)
@@ -957,7 +957,9 @@
     "plt.imshow(dic.find_zone(strain), cmap='jet')\n",
     "plt.clim(0.10,0.16)\n",
     "plt.axis('off')\n",
-    "plt.savefig('../data/Tension image strain box/2crop_png',dpi=300,bbox_inches='tight')\n"
+    "\n",
+    "\n",
+    "\n"
    ],
    "metadata": {
     "collapsed": false,
Index: .idea/project.iml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+><?xml version=\"1.0\" encoding=\"UTF-8\"?>\r\n<module type=\"PYTHON_MODULE\" version=\"4\">\r\n  <component name=\"NewModuleRootManager\">\r\n    <content url=\"file://$MODULE_DIR$\">\r\n      <excludeFolder url=\"file://$MODULE_DIR$/data\" />\r\n    </content>\r\n    <orderEntry type=\"inheritedJdk\" />\r\n    <orderEntry type=\"sourceFolder\" forTests=\"false\" />\r\n  </component>\r\n  <component name=\"PyDocumentationSettings\">\r\n    <option name=\"renderExternalDocumentation\" value=\"true\" />\r\n  </component>\r\n</module>
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/project.iml b/.idea/project.iml
--- a/.idea/project.iml	(revision b1065dd120cc4a9ccf0394348e79d07fc26cc293)
+++ b/.idea/project.iml	(date 1629859561974)
@@ -4,7 +4,7 @@
     <content url="file://$MODULE_DIR$">
       <excludeFolder url="file://$MODULE_DIR$/data" />
     </content>
-    <orderEntry type="inheritedJdk" />
+    <orderEntry type="jdk" jdkName="Python 3.8 (train) (2)" jdkType="Python SDK" />
     <orderEntry type="sourceFolder" forTests="false" />
   </component>
   <component name="PyDocumentationSettings">
Index: .idea/misc.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+><?xml version=\"1.0\" encoding=\"UTF-8\"?>\r\n<project version=\"4\">\r\n  <component name=\"JavaScriptSettings\">\r\n    <option name=\"languageLevel\" value=\"ES6\" />\r\n  </component>\r\n  <component name=\"ProjectRootManager\" version=\"2\" project-jdk-name=\"Python 3.8 (project)\" project-jdk-type=\"Python SDK\" />\r\n</project>
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/misc.xml b/.idea/misc.xml
--- a/.idea/misc.xml	(revision b1065dd120cc4a9ccf0394348e79d07fc26cc293)
+++ b/.idea/misc.xml	(date 1629859562058)
@@ -3,5 +3,5 @@
   <component name="JavaScriptSettings">
     <option name="languageLevel" value="ES6" />
   </component>
-  <component name="ProjectRootManager" version="2" project-jdk-name="Python 3.8 (project)" project-jdk-type="Python SDK" />
+  <component name="ProjectRootManager" version="2" project-jdk-name="Python 3.8 (train) (2)" project-jdk-type="Python SDK" />
 </project>
\ No newline at end of file
